src/pspm_sf_dcm.m

```diff
@@ -131,6 +132,26 @@
   '\n=========================================================\n'], c(4:6));
 if isfield(options, 'missing')
   ymissing = options.missing;
+  if isfile(ymissing)
+    [~, ~, fExt] = fileparts(ymissing);
+    switch lower(fExt)
+      case '.mat'
+        % A MAT file
+        ymissing = load(ymissing);
```

Comment:
Here, the generic function 'pspm_get_timing' should be used. 

Compare the corresponding code in pspm_dcm: pspm_get_timing('epochs', ...
      model.missing{iSn}, 'seconds');